knowledge_base/bundle_variables/README.md

```diff
@@ -0,0 +1,65 @@
+# bundle_variables
+
+Example to show how bundle variables work.
+
+We will focus on the behavior of the variables section alone. To do so, we exclusively use
+the `validate` command, and use its JSON output mode to focus on the variables section.
```

Comment:
```suggestion
This example shows how to validate the custom variables you have defined for your bundle using the `bundle validate` command and its JSON output mode.
```

Reply:
I think that is the main intent of this KB, but another sentence could also be added about showing how to set variable defaults.